k8s/Dockerfile

```diff
@@ -21,14 +22,15 @@ RUN go mod download
 RUN go build -a -o vineyardctl cmd/main.go && \
     strip vineyardctl || true
 
+
+#ENV base_image=${base_image}
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+FROM ${BASE_IMAGE}
```

Comment:
Do we need other base images?

Reply:
Yes, the image `gcr.io/distroless/static:nonroot` doesn't have the `mount` command and the csi driver image need it.

Reply:
Then you should use `gcr.io/distroless/base:debug` which has busybox installed and there would be a `mount` command available.

NEVER use your build environments (e.g., golang image) as the base to release your artifacts.